components/NewIngredientForm.js

```diff
@@ -0,0 +1,90 @@
+import { useState } from "react";
+import styled from "styled-components";
+
+const placeholderImageUrl =
+  "https://static.vecteezy.com/system/resources/previews/003/170/825/original/isolated-food-plate-fork-and-spoon-design-free-vector.jpg";
+
+const NewIngredientForm = ({ onAddIngredient, flavors }) => {
+  const [name, setName] = useState("");
+  const [flavor, setFlavor] = useState("");
+  const [imgUrl, setImgUrl] = useState("");
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
```

Comment:
please be explicit when naming arguments, parameters, functions and variables:

```suggestion
  const handleSubmit = (event) => {
    event.preventDefault();
```